pkg/networkservice/common/cleanup/options.go

```diff
@@ -24,8 +24,8 @@ type options struct {
 // Option - options for the cleanup chain element
 type Option func(*options)
 
-// WithCCClose - closes clientconn to prevent calling requests/closes on other datapath objects
-func WithCCClose() Option {
+// WithoutGrpcCall - closes client connection to prevent calling requests/closes on other endpoints
+func WithoutGrpcCall() Option {
```

Comment:
Could you add `GRPC` into https://github.com/networkservicemesh/sdk/blob/main/staticcheck.conf and fix the linter issue?

Reply:
Sure! Done